scripts/release.sh

```diff
@@ -13,64 +13,70 @@ if [[ $# != 2 ]] && [[ $# != 3 ]]; then
     echo 'then this script runs "make deploy" after sourcing "environment.{DEST}",'
     echo "where DEST is the destination branch."
     echo
-    echo "If the --force flag is given, deployment will proceed even if CI checks fail."
+    echo "If ${DSS_FORCE_RELEASE}=='force', deployment will proceed even if CI checks fail."
```

Comment:
I am worried about the use of env vars to drive something as critical as this. Env vars are too implicit to toggle top level flags. I think it's preferable to call this script with `--force` in the caller's location.

Reply:
That's reasonable, reverted to `--force`.

However, another mechanism will be needed if we wish to force promotion through the build box,